src/Command/Validators.php

```diff
@@ -61,15 +49,9 @@ public static function validateEntityName($shortcut)
     }
 
     /**
-     * @static
-     *
-     * @param string $class
-     *
      * @throws \InvalidArgumentException
-     *
-     * @return string
      */
-    public static function validateClass($class)
+    public static function validateClass(string $class): string
```

Comment:
input class-string and outputs regular string?

Reply:
It can be called with any string, since the input is given by the user, not only class-string.

If you typehint `class-string` phpstan will say that
```
if (!class_exists($class)) {
```
is useless.